src/dcqc/target.py

```diff
@@ -27,42 +28,18 @@ class Target(SerializableMixin):
             Defaults to None.
     """
 
-    type: str
-    id: Optional[str]
     files: list[File]
+    id: Optional[str]
+    type: str
 
     def __init__(self, *files: File, id: Optional[str] = None):
```

Comment:
Good eye. The only reason is that I want the constructor interface to use `*files` instead of `files`, negating the need to wrap a single file in a list for constructing a target. Alternatively, I could add some special logic for auto-wrapping single files in a list, but that involves having a `File | list[File]` type hint, which I'm not a fan of. Another option is to create a class method that implements this constructor interface. I'll think about this some more and potentially refactor.

Reply:
Makes sense to me!